src/ngraph_builder.cc

```diff
@@ -52,7 +52,7 @@ static bool VecStrCmp(const std::vector<string>& a,
 }
 
 static Status ValidateInputCount(const Node* op, size_t count) {
-  if (op->num_inputs() != count) {
+  if ((unsigned)op->num_inputs() != count) {
```

Comment:
Change the signature of this function to replace `size_t` to match that of the Node::num_inputs() which is `tensorflow::int32`.
On the top of the file use: `using tensorflow::int32`.